website/static/js/remote/knockout.remote.js

```diff
@@ -59,9 +59,17 @@ function onWsClose() {
 
 function wsActionUpdatePlaylistTrack(newPlaylistTracks) {
   mappedPlaylistTracks = $.map(newPlaylistTracks, function(item) {
```

Comment:
Don't rly want to as AJAX convert the JSON sent by the server to an Object and Javascript can't map objects, map is a method of Arrays.
If I want to work with pure JS (why ? All the front run with Knockout running itself with Jquery...) I'll need to convert that object to an array and then map it (or filter it, same deal).

The point of using jQuery's map is that jQuery CAN map from Objects :)